google-cloud-core/src/com/google/cloud/tools/intellij/project/ProjectSelectionDialog.java

```diff
@@ -210,8 +209,9 @@ public void onFailure(Throwable throwable) {
                     String errorMessage =
                         Optional.ofNullable(throwable.getMessage())
                             .orElse(throwable.getClass().getName());
-                    dialogWrapper.setErrorInfoAll(
-                        Collections.singletonList(new ValidationInfo(errorMessage)));
+                    //                    dialogWrapper.setErrorInfoAll(
```

Comment:
comment got collapsed after reformatting. adding again:

@ivanporty you are closer to this code - do you recall what exactly #setErrorInfoAll is doing and if there is an alternative?

In the eap, this method was made final, so we can no longer implement it in the DialogWrapper (ProjectSelectionDialogWrapper inner class). I commented it out here just to get things to compile